ruby-challenges.rb

```diff
@@ -10,18 +10,53 @@
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
+def find_words_containing_letter(beverages_array, letter)
+    beverages_array.select { |word| word.include?(letter)}
+end
+puts find_words_containing_letter(beverages_array, letter_o)
```

Comment:
Awesome start on this one but is your current output the expected output? How can we return that array that we are expecting within the method?